Rakefile

```diff
@@ -262,6 +263,399 @@ task :xcode => [:dependencies] do
+                  "gs" => "ghostscript",
+      }
+      developer_tools = {"convert" => "imagemagick",
+                        "gs" => "ghostscript",
+                        "sentry-cli" => "getsentry/tools/sentry-cli",
+                        "gpg" => "gpg",
+                        "git-crypt" => "git-crypt",
+      }
+
+      #Check for tool, install if not installed
+      task :check_oss do
+        tool_check(oss_tools)
+      end
+
+      task :check_developer do
+        tool_check(developer_tools)
+      end
+
+      #check machine for if the developer tool is present, if not install
+      def tool_check(hash)
+        hash.each do |key, value|
+          puts "Checking system for #{key}"
+          unless command?(key)
+            tool_install(value)
+          else
+            puts "#{key} found"
+          end
+        end
+      end
+
+      ##install selected developer tool
+      def tool_install(tool)
+        puts "#{tool} not found.  Installing #{tool}"
+        sh "brew install #{tool}"
+      end
+
+    #End namespace addons
+    end
+
+  #End namespace Tools
+  end
+
+  namespace :lint do
+    task :check do
+      if !git_initialized?
+        puts "Initializing git repository"
+        sh "git init", verbose: false
+      end
+
+      Rake::Task["git:install_hooks"].invoke
+    end
+
+    def git_initialized?
+      sh "git rev-parse --is-inside-work-tree > /dev/null 2>&1", verbose: false
+    end
+  end
+
+#End namespace install
+end
+
+#Credentials deals with the setting up the developer's WPCOM API app ID and app Secret
+namespace :credentials do
+  task :setup => %w[credentials:prompt credentials:set_app_secrets]
+
+  task :prompt do
+    puts ""
+    puts "====================================================================================="
+    puts "To be able to log into the WordPress app while developing you will need to setup API credentials"
+    puts "To do this follow these steps"
+    puts ""
+    puts ""
+    puts ""
+    puts "====================================================================================="
+
+    puts "1. Create a WordPress.com account at https://wordpress.com/start/user (if you don't already have one)."
+    prompt_for_continue
+
+    puts "====================================================================================="
+    puts "2. Create an application at https://developer.wordpress.com/apps/."
+    prompt_for_continue
+
+    puts "====================================================================================="
+    puts '3. Set "Redirect URLs"= https://localhost and "Type" = Native and click "Create" then "Update".'
+    prompt_for_continue
+
+    puts "====================================================================================="
+    puts "4. Copy the Client ID and Client Secret from the OAuth Information."
+    prompt_for_continue
+  end
+
+  def prompt_for_continue
+    STDOUT.puts "Please press enter to continue"
```

Comment:
**Nitpick** This text will be read by users that are expected to do something _and only then_ press enter to continue.

How can we make this clearer?

Reply:
Added some more details to make what the script is waiting for to the developer running it